spring-data-microservice-transaction-sample/README.md

```diff
@@ -0,0 +1,548 @@
+# Microservice Transaction Sample
+
+This tutorial describes how to create a sample Spring Boot application for microservices by using Spring Data JDBC for ScalarDB with Two-phase Commit Transactions of ScalarDB.
+
+For details, please see [Two-phase Commit Transactions in ScalarDB](https://github.com/scalar-labs/scalardb/tree/master/docs/two-phase-commit-transactions.md) and [Guide of Spring Data JDBC for ScalarDB](https://github.com/scalar-labs/scalardb-sql/blob/main/docs/spring-data-guide.md).
+
+## Prerequisites
+
+- Java (OpenJDK 8 or higher)
+- Gradle
+- Docker, Docker Compose
+
+In addition, you need access to the [ScalarDB SQL GitHub repository](https://github.com/scalar-labs/scalardb-sql) and [Packages in ScalarDB SQL repository](https://github.com/orgs/scalar-labs/packages?repo_name=scalardb-sql).
+These repositories are available only to users with a commercial license and permission.
+To get a license and permission, please [contact us](https://scalar-labs.com/contact_us/).
+
+You also need the `gpr.user` property for your GitHub username and the `gpr.key` property for your personal access token.
+You must either add these properties in `~/.gradle/gradle.properties` or specify the properties by using the `-P` option when running the `./gradlew` command as follows:
+
+```shell
+$ ./gradlew run ... -Pgpr.user=<YOUR_GITHUB_USERNAME> -Pgpr.key=<YOUR_PERSONAL_ACCESS_TOKEN>
+````
+
+Or you can also use environment variables, `USERNAME` for your GitHub username and `TOKEN` for your personal access token.
+
+```shell
+$ export USERNAME=<YOUR_GITHUB_USERNAME>
+$ export TOKEN=<YOUR_PERSONAL_ACCESS_TOKEN>
+```
+
+For more details, please see [Install - ScalarDB SQL](https://github.com/scalar-labs/scalardb-sql#install).
+
+## Sample application
+
+### Overview
+
+This tutorial describes how to create a Spring Boot sample application for Microservice Transaction for the same use case as [ScalarDB Sample](https://github.com/scalar-labs/scalardb-samples/tree/main/scalardb-sample) but by using Two-phase Commit Transactions in ScalarDB.
+
+There are two microservices called the *Customer Service* and the *Order Service* based on the [*Database-per-service* pattern](https://microservices.io/patterns/data/database-per-service.html) in this sample application.
+
+The Customer Service manages customers' information including credit card information like a credit limit and a credit total.
+The Order Service is responsible for order operations like placing an order and getting order histories.
+Each service has gRPC endpoints. Clients call the endpoints, and the services call the endpoints each other as well.
+The Customer Service and Order Service use MySQL and Cassandra through ScalarDB, respectively.
```

Comment:
@komamitsu Sorry, I think I added `The` for all.

@josh-wong Are you removing `The` because we treat them as proper nouns?

Reply:
@feeblefakie After some research into how these types of services are referred to, most organizations seem to use `the` (with variations in how they capitalize the services). I've reverted my changes and added `the` in places where it should be added throughout this document. 

> Are you removing `The` because we treat them as proper nouns?

It's a bit tricky. For example, when referring to a department within a company called `Customer Service`, we wouldn't call that department `the Customer Service` (unless it modified a noun, like `the Customer Service department`).